src/utils/clipboard.ts

```diff
@@ -0,0 +1,9 @@
+export const copyToClipboard = async (code: string): Promise<boolean> => {
+  try {
+    await navigator.clipboard.writeText(code);
+    return true;
+  } catch (err) {
+    console.error("Failed to copy to clipboard", err);
+    return false;
+  }
+};
```

Comment:
lets use this here too? https://github.com/transmissions11/flux/blob/1ccbfd3e7cd8d469526dd69754be8059f7e866c9/src/components/App.tsx#L759

Reply:
Moved copyMessagesToClipboard to /utils/clipboard.ts